@types/interfaces.d.ts

```diff
@@ -0,0 +1 @@
+export type ScrollerScrollDirection = 'horizontal' | 'vertical' | 'none' | null | undefined;
```

Comment:
Added component name prefix similarly to the `orientation` types on other components, although it looks a bit weird here.

Reply:
I think it's ok so we don't have to think too much about the naming of these. These TS defs are kind of temporary anyway since next gen will be natively TS.